lib/api.es6.js

```diff
@@ -8,7 +8,7 @@ const SCOPES = 'history,identity,mysubreddits,read,subscribe,vote,submit,' +
                'modwiki,privatemessages,report,wikiedit,wikiread';
 
 
-const login = (apiOptions, username, pass) => {
+const login = (apiOptions, username, pass, orderedHeaders, clientUserAgent) => {
```

Comment:
I chose _not_ to make it optional on purpose since everyone should be sending these as general good practice. I was planning on making this a major version upgrade as well since this is an api breaking change.  I'm up for discussion on both of those things.